client/extensions/woocommerce/app/settings/payments/payment-method-item.js

```diff
@@ -91,50 +88,11 @@ class PaymentMethodItem extends Component {
 				payment_method: method.id,
 			} );
 		}
-
-		const successAction = () => {
-			return successNotice(
-				translate( 'Payment method successfully saved.' ),
-				{ duration: 4000 }
-			);
-		};
-
-		const errorAction = () => {
-			this.props.changePaymentMethodEnabled(
-				site.ID,
-				method.id,
-				! enabled
-			);
-			return errorNotice(
-				translate( 'There was a problem saving the payment method. Please try again.' )
-			);
-		};
-		this.props.savePaymentMethodEnabled(
-			site.ID,
-			method.id,
-			e.target.value,
-			successAction,
-			errorAction
-		);
 	}
 
 	onSave = () => {
```

Comment:
Since this doesn't save anymore, maybe it should be renamed... `onClose`, `onDone`... something like that? I guess you'd have to rename the prop in each PaymentMethod component, too. I'll leave this up to you :)